ebuild-writing/file-format/text.xml

```diff
@@ -120,18 +120,27 @@ EAPI 7 version commands</uri> may be used to manipulate and extract
 ebuild version components.
 </p>
 
+<p>
+The formal specification of version
+<uri link="https://projects.gentoo.org/pms/latest/pms.html#x1-250003.2">format
+</uri> and the comparison
+<uri link="https://projects.gentoo.org/pms/latest/pms.html#x1-260003.3">
+algorithm</uri> can be found in PMS.
```

Comment:
Again, `8` instead of `latest` because TeX4ht doesn't provide immutable fragment identifiers. (I'm going to investigate if anything can be done about this.)

Also, could you put the word `format` on the next line? I guess it's within the guidelines as-is, but the `</uri>` at the beginning of the next line looks strange. :)

Reply:
Understood! Yes, it's consistent this way too.